client/service/src/config.rs

```diff
@@ -93,6 +93,8 @@ pub struct Configuration {
 	pub rpc_cors: Option<Vec<String>>,
 	/// RPC methods to expose (by default only a safe subset or all of them).
 	pub rpc_methods: RpcMethods,
+	/// Maximum payload of rpc request/responses.
+	pub rpc_max_payload_override: Option<usize>,
```

Comment:
Same here, I think `override` adds nothing. I'd prefer just `rpc_max_payload`, or possibly `set_rpc_max_payload` is there's confusion with the getter.